pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/cleanup/SchemaCleanupTaskStatelessTest.java

```diff
@@ -97,6 +97,7 @@ public void teardown() {
   public void testSchemaCleanupTask()
       throws Exception {
     PinotMetricUtils.cleanUp();
+    PinotMetricUtils.getPinotMetricsRegistry();
```

Comment:
What we usually do is to call `ControllerMetrics.register(mock(ControllerMetrics.class));` in the test set up annotated as `@BeforeClass`. You may find more examples if you check the usage of `ServerMetrics.register()`

Reply:
I don't think the failed test is because ControllerMetrics is uninitialized, but because of ControllerMetrics needs to call PinotMetricUtils when emitting metrics and PinotMetricUtils was cleaned up at the beginning of the test. I can also delete `PinotMetricUtils.cleanUp();` but I am not sure if that line is useful for other purpose.

This is where NPE was thrown
https://github.com/apache/pinot/blob/4eeae1197d9f70b4c33e7220c0dc3a073e783648/pinot-common/src/main/java/org/apache/pinot/common/metrics/AbstractMetrics.java#L340